src/app/modals/add-to-group/add-to-group.component.ts

```diff
@@ -341,6 +345,8 @@ export class AddToGroupModal implements OnInit, OnDestroy, AfterViewInit {
           })
           .catch( error => {
             console.error(error)
+            this.error.recentGroups = true
+            this.loading.recentGroups = false
```

Comment:
I looks like we have a lot of different situations where we set `this.loading.recentGroups = false`. Wondering if we could consolidate all of these into one place somehow. Seems like we want to stop loading recent groups regardless of the fact if it errors out or processes successfully. Maybe there is a way we can clean up this observable?

Reply:
As we discussed offline looks like this might be a bigger endeavor than we thought. I created PLATO-192 for taking a closer look at this logic and learning more about observables